keras/src/quantizers/gptqconfig.py

```diff
@@ -0,0 +1,70 @@
+from absl import logging
+
+from keras.src.api_export import keras_export
+from keras.src.quantizers.gptqutils import quantize_model
+
+
+@keras_export(["keras.GPTQConfig", "keras.quantizers.GPTQConfig"])
+class GPTQConfig:
+    """
+    Configuration class for the GPTQ (Generative Pre-trained Transformer
+    Quantization) algorithm.
+
+    This class holds all the parameters needed to apply the GPTQ method
+    to a model. Its attributes are based on the original command-line
+    arguments from  the research repository's `opt.py` script.
+
+    Args:
+        dataset (str): Path to the calibration dataset.
+        wbits (int, optional): The number of bits to quantize the weights to.
+            Defaults to 4.
+        nsamples (int, optional): The number of calibration data samples to
+                                  use. Defaults to 128.
+        seqlen (int, optional): The sequence length to use for calibration.
+            Defaults to 512.
+        percdamp (float, optional): The percentage of Hessian damping to use.
+            Defaults to 0.01.
+        groupsize (int, optional): The size of the group of weights to
+                                    quantize together.A groupsize of
+                                    -1 means quantization is done per-column.
+                                     Defaults to 128.
+        symmetric (bool, optional): If True, uses symmetric quantization.
+                                    If False,uses asymmetric quantization.
+                                    Defaults to False.
+        act_order (bool, optional): If True, quantizes columns in order of
+                                    decreasing activation size.
+                                    Defaults to False.
+    """
+
+    def __init__(
+        self,
+        dataset,
+        tokenizer: str,
```

Comment:
tokenizer - should be a callable right? not a string

Reply:
Yes you are right, updated this